examples/nidaqmx_analog_input/teststand_nidaqmx.py

```diff
@@ -49,8 +49,11 @@ def destroy_nidaqmx_tasks() -> None:
         session_management_client = SessionManagementClient(
             discovery_client=discovery_client, grpc_channel_pool=grpc_channel_pool
         )
+        # Timeout is added when destroying sessions to ensure that if a cancellation is performed
+        # for the measurement, session is closed within an appropriate timeframe.
         with session_management_client.reserve_all_registered_sessions(
             instrument_type_id=INSTRUMENT_TYPE_NI_DAQMX,
+            timeout=60,
```

Comment:
Also, I would like to know if we could reduce the timeout to 30 seconds or even 10 seconds if that is sufficient. I knew that earlier we had a 60-second timeout but at least I'm unaware of the decision behind it.